app/views/horses/index.erb

```diff
@@ -11,10 +11,10 @@
   </tr>
   <% @horses.each do |horse| %>
     <tr>
-      <td><%= horse.name %></td>
-      <td><%= horse.breed.name %></td>
-      <td><%= horse.total_winnings %></td>
-      <td><%= horse.jockey.name %></td>
+      <td id="name"><%= horse.name %></td>
```

Comment:
Interesting. I think if you run this code it will result in multiple cells with the same id, which is no good in HTML. Would change it to a class instead. HOWEVER, would recommend being careful adding classes to our HTML to help us find items in our tests. Generally you'll want to get a little more creative with your CSS selectors in the test file. Take a look at `nth-child()`, or `nth-of-type()` if you want to really narrow things down.